setup.py

```diff
@@ -5,6 +5,19 @@
     version="0.1.0",
     author="Andreas Doerr",
     author_email="andreasdoerr@gmx.net",
+    license='MIT',
     description="Tool to automatically query impfterminservice.de",
     packages=setuptools.find_packages(),
+    python_requires='>=3.6',
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+    ],
```

Comment:
metadata for pypi. you can't make a package from this yet due to the way example.py needs editing, but maybe later...